requirements.txt

```diff
@@ -0,0 +1,67 @@
+# All dependencies needed to run rucio client (and server/daemons) should be defined here
```

Comment:
Just a quick question, did you update any of the requirements here, or is this the state of the etc/pip* files?

Reply:
I removed `setuptools` (since we don't need it for the actual code and it is obvious for the setup.py files), but otherwise only the extras were added.